examples/notebooks/beam-ml/image_processing_beam.ipynb

```diff
@@ -0,0 +1,933 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "source": [
+        "# @title ###### Licensed to the Apache Software Foundation (ASF), Version 2.0 (the \"License\")\n",
+        "\n",
+        "# Licensed to the Apache Software Foundation (ASF) under one\n",
+        "# or more contributor license agreements. See the NOTICE file\n",
+        "# distributed with this work for additional information\n",
+        "# regarding copyright ownership. The ASF licenses this file\n",
+        "# to you under the Apache License, Version 2.0 (the\n",
+        "# \"License\"); you may not use this file except in compliance\n",
+        "# with the License. You may obtain a copy of the License at\n",
+        "#\n",
+        "#   http://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing,\n",
+        "# software distributed under the License is distributed on an\n",
+        "# \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n",
+        "# KIND, either express or implied. See the License for the\n",
+        "# specific language governing permissions and limitations\n",
+        "# under the License"
+      ],
+      "metadata": {
+        "id": "NsNImDL8TGM1"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Image Processing using Apache Beam\n",
```

Comment:
Could you please add colab/github links like our other notebooks, for example:

https://github.com/apache/beam/blob/c2de3391bce1165e0e835b1475cfb254382cdad0/examples/notebooks/beam-ml/automatic_model_refresh.ipynb#L52

<img width="526" alt="image" src="https://github.com/apache/beam/assets/42773683/ec9a9ad5-b40f-49f1-927a-df0b27059228">

Note that the links should go to apache/beam master's version of these on colab/github even though those links won't exist until the PR is merged